config/options.go

```diff
@@ -561,6 +564,11 @@
 			return fmt.Errorf("config: bad forward-auth-url %s : %w", o.ForwardAuthURLString, err)
 		}
 		o.ForwardAuthURL = u
+		switch o.ForwardAuthType {
+		case ForwardingProxyNginx, ForwardingProxyTraefik:
```

Comment:
I think we can now remove this check here because it will be called in the service itself, where it makes more sense to validate it?

Reply:
Fixed